### PR TITLE
https://explore-labyrinth.netlify.app/

### PR DESCRIPTION
**Pull request by Sherin-Susan-Thomas**
Project-Labyrinth By Sherin and Justine

### CODE REVIEW

File: code/src/components/Game.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  align-items: center;
+`;
+
+const Game = () => {
+  const gameStatus = useSelector((store) => store.labyrinth.gameStatus);
+  console.log(gameStatus);
+  return <Main>{gameStatus ? <Gamepage /> : <Start />}</Main>;
```
Comment (emmajosefina) [inline code/src/components/Game.js:18]: Nice turnery operator!

File: code/src/components/Gamepage.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        <div className="content">
+          {actions.length !== 0 && (
+            <>
+              <div className="bordered">
+                <h3 className="nes-container is-rounded is-dark">
+                  {description}
+                </h3>
+              </div>
+              <div className="position-wrapper">
+                <h3 className="position-text">Coordinates:</h3>
```
Comment (emmajosefina) [inline code/src/components/Gamepage.js:51]: Nice that you added this!

File: code/src/components/Start.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+//import styled from "styled-components";
```
Comment (emmajosefina) [inline code/src/components/Start.js:3]: You can delete this if you don't use it :)))

File: code/src/index.css [hunk 4]
```diff
@@ -1,13 +1,87 @@
...
+  font-family: "Fira Mono", monospace;
+  font-family: "Press Start 2P", cursive;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: "Press Start 2P", cursive;
```
Comment (emmajosefina) [inline code/src/index.css:12]: Love this font!

**Review by emmajosefina — APPROVED** [review]
Really great job this week, Sherin and Justine! 🥇 

I don't know what you think about Redux and this labyrinth project, but I found it really had to grasp the concept! I will give it sometime. You on the other hand seem like you know it 💯 % already! Nice and clean code like always and I love your design!

File: code/package.json [hunk 5]
```diff
@@ -3,16 +3,20 @@
...
   "dependencies": {
+    "@reduxjs/toolkit": "^1.8.1",
     "babel-eslint": "^10.0.3",
     "eslint": "^6.7.1",
     "eslint-config-airbnb": "^18.0.1",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.16.0",
     "eslint-plugin-react-hooks": "^2.3.0",
+    "nes.css": "^2.3.0",
```
Comment (jiiinCho) [inline code/package.json:14]: It is super cool library and it's great you tried it, I really like the components and images that you used in the app! :)

File: code/public/index.html [hunk 6]
```diff
@@ -13,7 +12,21 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
+    <link
+      href="https://unpkg.com/nes.css@latest/css/nes.min.css"
+      rel="stylesheet"
+    />
+    <style>
```
Comment (jiiinCho) [inline code/public/index.html:19]: Is it a default setting from styled-component?

File: code/src/components/Game.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+const Main = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  align-items: center;
+`;
+
+const Game = () => {
+  const gameStatus = useSelector((store) => store.labyrinth.gameStatus);
+  console.log(gameStatus);
```
Comment (jiiinCho) [inline code/src/components/Game.js:17]: maybe this console can be removed?!

File: code/src/components/Game.js (hunk 1 above)
Comment (jiiinCho) [reply code/src/components/Game.js:18]: agree!

File: code/src/components/LoadingIndicator.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+`;
+
+const LoadingIndicator = () => {
+  return (
+    <Main>
+      <div className="loading-container">
+        <div className="loading-img-container">
+          <img
+            className="loading-img"
+            src="https://i.ibb.co/ZKSBwtC/Untitled-design-2.gif"
```
Comment (jiiinCho) [inline code/src/components/LoadingIndicator.js:18]: super cool loading indicator! where do you get the .gif source?!

File: code/src/components/Start.js [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+        value={input}
+        onChange={(event) => setInput(event.target.value)}
+        placeholder="enter your name and step into the labyrinth"
+      />
+      <div className="button-container">
+        <button
+          className="nes-btn is-warning"
+          disabled={!input}
+          onClick={() => {
+            inputAdd();
```
Comment (jiiinCho) [inline code/src/components/Start.js:40]: Daniel said it is better to use onClick={() => inputAdd()}, not sure the reason correctly though 😅

File: code/src/components/Gamepage.js [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+              </div>
+            </>
+          )}
+          {actions.length === 0 ? (
+            <div>
+              <End />
+            </div>
+          ) : (
+            actions.length > 0 &&
+            actions.map((item) => <GameCard key={item.direction} {...item} />)
```
Comment (jiiinCho) [inline code/src/components/Gamepage.js:72]: cool that you used spread operator to pass props, clever!

**Review by jiiinCho — COMMENTED** [review]
It was super great project that you used Nes library recommended by Masky and found images/gifs according to the design concept. Super impressive that you used ternary & spread operator in clever way. You both are such a pro! :)

**Pull request closed without merging**